### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by jakexks**
Addresses #16 and #18

### CODE REVIEW

File: bundle/cert-manager-operator/1.1.0/cert-manager-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+              app: cert-manager-operator
+          template:
+            metadata:
+              labels:
+                app: cert-manager-operator
+            spec:
+              serviceAccountName: cert-manager-operator
+              containers:
+              - name: cert-manager-operator
+                image: registry.connect.redhat.com/jetstack/cert-manager-operator:v1.1.0-ubi
```
Comment (meyskens) [inline bundle/cert-manager-operator/1.1.0/cert-manager-operator.v1.1.0.clusterserviceversion.yaml:55]: Might be worth double checking i think they changed this

File: watches.yaml [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
     webhook.image.repository: registry.connect.redhat.com/jetstack/cert-manager-webhook
-    webhook.image.tag: v0.16.0-ubi
+    webhook.image.tag: v1.1.0-ubi
     cainjector.image.repository: registry.connect.redhat.com/jetstack/cert-manager-cainjector
-    cainjector.image.tag: v0.16.0-ubi
+    cainjector.image.tag: v1.1.0-ubi
     installCRDs: true
-    extraArgs[0]: --acme-http01-solver-image=registry.connect.redhat.com/jetstack/cert-manager-acmesolver:v0.16.0-ubi
\ No newline at end of file
+    extraArgs[0]: --acme-http01-solver-image=registry.connect.redhat.com/jetstack/cert-manager-acmesolver:v1.1.0-ubi
```
Comment (meyskens) [inline watches.yaml:14]: Missing NL

Comment (jakexks) [reply watches.yaml]: There wasn't one before! Fixed

File: bundle/cert-manager-operator/1.1.0/certmanagers.operator.cert-manager.io.crd.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    listKind: CertManagerList
+    plural: certmanagers
+    singular: certmanager
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      type: object
+      x-kubernetes-preserve-unknown-fields: true
```
Comment (meyskens) [inline bundle/cert-manager-operator/1.1.0/certmanagers.operator.cert-manager.io.crd.yaml:18]: Is this still needed?

Comment (jakexks) [reply bundle/cert-manager-operator/1.1.0/certmanagers.operator.cert-manager.io.crd.yaml]: Probably not, openshift is already on k8s 1.19

**Pull request merged**